rosbag2_py/src/rosbag2_py/format_bag_metadata.cpp

```diff
@@ -139,13 +140,37 @@ void format_topics_with_type(
       info_stream << std::endl;
     };
 
+  std::vector<size_t> sorted_idx(topics.size());
+  std::iota(sorted_idx.begin(), sorted_idx.end(), 0);
+  std::sort(
+    sorted_idx.begin(),
+    sorted_idx.end(),
+    [&topics, sort_method](size_t i1, size_t i2) {
+      if (sort_method == "type")
+      {
+        return topics[i1].topic_metadata.type < topics[i2].topic_metadata.type;
+      }
+      if (sort_method == "count")
+      {
+        return topics[i1].message_count < topics[i2].message_count;
+      }
+      if (sort_method == "serialization_format")
```

Comment:
I am not sure about this use case. most likely the serialization format could be the same in the single bag file which is `cdr`?